hips/draw/simple.py

```diff
@@ -117,6 +117,30 @@ def _fetch_tiles(self) -> HipsTile:
             tile = HipsTile.fetch(tile_meta, url)
             yield tile
 
+    @staticmethod
+    def _measure_tile_shape(corners: tuple) -> List[list]:
+        """Compute tile edges and diagonals."""
```

Comment:
"Compute tile edges and diagonals" -> "Compute length of tile edges and diagonals"